cdap-app-fabric/src/test/java/co/cask/cdap/metadata/MetadataSubscriberServiceTest.java

```diff
@@ -439,6 +448,56 @@ public void testProfileMetadataWithNoProfilePreferences() throws Exception {
     }
   }
 
+  @Test
+  public void testAppDeletionMessage() throws Exception {
```

Comment:
Does this test fail without the above changes ?

Reply:
Yes, if you use new Gson() in the AdminEventPublisher, the test will fail